internal/server/httpServer.go

```diff
@@ -0,0 +1,127 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/charmbracelet/log"
+	"github.com/dicedb/dice/config"
+	"github.com/dicedb/dice/internal/clientio"
+	"github.com/dicedb/dice/internal/ops"
+	"github.com/dicedb/dice/internal/querywatcher"
+	"github.com/dicedb/dice/internal/server/utils"
+	"github.com/dicedb/dice/internal/shard"
+	dstore "github.com/dicedb/dice/internal/store"
+)
+
+type HTTPServer struct {
+	queryWatcher *querywatcher.QueryManager
+	shardManager *shard.ShardManager
+	ioChan       chan *ops.StoreResponse
+	watchChan    chan dstore.WatchEvent
+	httpServer   *http.Server
+}
+
+// NewHTTPServer
+// TODO: This isn't ideal as we create a separate instance of ShardManager, which we want to be 1, which all servers share
```

Comment:
Is this comment outdated? Aren't we already creating and sharing a common shard manager across both servers?

Reply:
Yes, correct. This is outdated.